### DISCUSSION TITLE
Getting Started

### DISCUSSION THREAD

**Discussion by Alan-Python** [yattee/yattee#844] [Q&A]
So I'm completely new to Yattee, so how to get started using Privacy-Focused Yattee. I'm tired of same old YouTube ads

**Comment by unional** [comment]
Consolidating this RFC into the canonical activation thread: https://github.com/agentskills/agentskills/issues/57#issuecomment-4540026815

Key revision from #408: drop `metadata.activation`; use top-level `activation.lifecycle` (merged with @venikman signals/context/relevance model).
